.github/workflows/backport.yml

```diff
@@ -0,0 +1,54 @@
+name: Backport PR to branch
+on:
+  issue_comment:
+    types: [created]
+
+jobs:
+  backport:
+    if: github.event.issue.pull_request != '' && contains(github.event.comment.body, '/backport to')
+    runs-on: ubuntu-20.04
+    steps:
+    - name: Extract backport target branch
+      uses: actions/github-script@v3
+      id: target-branch-extractor
+      with:
+        result-encoding: string
+        script: |
+          if (context.eventName !== "issue_comment") throw "Error: This action only works on issue_comment events.";
+
+          // extract the target branch name from the trigger phrase containing these characters: a-z, A-Z, digits, forward slash, dot, hyphen, underscore
+          const regex = /\/backport to ([a-zA-Z\d\/\.\-\_]+)/;
+          target_branch = regex.exec(context.payload.comment.body);
+          if (target_branch == null) throw "Error: No backport branch found in the trigger phrase.";
```

Comment:
Suggest enforcing `/\/backport to release\/\d\.\d(-preview\d)?/` to avoid garbage `\backport to freddy` PRs.

Nit: could just ignore poorly-formed comments i.e. anything that doesn't match the full regular expression.